src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Node/JsonArrayConverter.cs

```diff
@@ -34,10 +34,20 @@ public override void Write(Utf8JsonWriter writer, JsonArray? value, JsonSerializ
             }
         }
 
-        public static JsonArray ReadList(ref Utf8JsonReader reader, JsonNodeOptions? options = null)
+        public static JsonArray ReadList(ref Utf8JsonReader reader, JsonNodeOptions options)
         {
-            JsonElement jElement = JsonElement.ParseValue(ref reader);
-            return new JsonArray(jElement, options);
+            if (options.PropertyNameCaseInsensitive)
```

Comment:
This appears to always validate for duplicates. Shouldn't it be predicated on `JsonSerializerOptions.AllowDuplicateProperties`?